### PR TITLE
Feature/npm release setup

### PR DESCRIPTION
**Pull request by linasi**
Setup build, test and release tasks.

### CODE REVIEW

**Review by ragnika — APPROVED** [review]

**Comment by ragnika** [comment]
👍

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/hydrogen # 18
+          cache: 'npm'
+          check-latest: true
```
Comment (vijayshiyani) [inline .github/workflows/build.yml:27]: should we fix the node version?

**Review by vijayshiyani — APPROVED** [review]
👍

**Comment by vijayshiyani** [comment]
merging it as, `sd-jwt-vc` library need to reference npm package

File: .github/workflows/build.yml (hunk 1 above)
Comment (linasi) [reply .github/workflows/build.yml:27]: what is wrong with the node version?

**Pull request merged**